modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/components/RuleList/RuleList.es.js

```diff
@@ -17,7 +17,6 @@ import 'clay-button';
 import 'clay-label';
 
 import 'clay-dropdown';
-import ClayTooltip from 'clay-tooltip';
```

Comment:
I was going to say this should also be removed from `dependencies` in the `package.json` of the module, but a quick check reveals it isn't even declared 🙈

Reply:
yeah I thought I'd find it in there too